packages/dialtone-css/lib/build/less/dialtone-reset.less

```diff
@@ -186,6 +186,7 @@ optgroup,
 select,
 textarea {
   margin: 0; /* 2 */
+  color: inherit;
```

Comment:
Select and textarea in some areas were rendering with browser native color, which no one would really notice.